crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -1469,6 +1469,12 @@ impl VisitAstPath for Analyzer<'_> {
         ident: &'ast Ident,
         ast_path: &mut AstNodePath<AstParentNodeRef<'r>>,
     ) {
+        if matches!(
+            ast_path.last(),
+            Some(AstParentNodeRef::MemberProp(_, MemberPropField::Ident))
+        ) {
+            return;
+        }
```

Comment:
I feel like this should visit `expr` and match on `Expr::Ident` instead (allowlisting instead of blocklisting)